pages/linux/at.md

```diff
@@ -0,0 +1,15 @@
+# at
+
+> Executes commands at a specified time.
+
+- Open `at` prompt to execute commands at specific time. When you're done, press Ctrl+D to save the schedule:
```

Comment:
I'd say

```markdown
- Open an `at` prompt to create a new schedule of commands; press Ctrl+D to save and exit:
```

Note the `an`. It's easy to read the `at` as part of the sentence, making it necessary to re-read until it clicks in your head that it's not.

I also think we can rely on the word `schedule` to signify that the commands will be specified `at a specific time`.